tests/selftest.intern.js

```diff
@@ -16,6 +16,11 @@ define({
 
 	maxConcurrency: 2,
 	tunnel: 'SauceLabsTunnel',
+	
```

Comment:
I needed to add the `loaders` config here to make the tests run with the `dojo-loader` otherwise I couldn't prove that it was working